doc/source/metadata/Principles.rst

```diff
@@ -0,0 +1,208 @@
+focussed on these aspects and not try to stretch it to cover 'who'.
+
+An example is CF cell methods, which are intended to cover 'what'
+(i.e. to ensure that data can be correctly interpreted) 
+but can unwittingly be extended into 'who'
+(e.g. to try to describe the post-processing that have been applied), 
+which can complicate its main purpose. 
+
+.. note::
+
+    **Metadata Categorization** 
+
+    To inform decisions about the implementation of metadata, 
+    it can be helpful to consider three types of metadata in terms
+    of what they are trying to achieve:
+
+    * **Where** - location in space and time,
+      and additional 'data space' information
+
+    * **What** - what is this value?
+      i.e. phenomenon, units, extra characteristics
+
+    * **Who** - data identity,
+      i.e. where it came from, provenance, data processing, etc
+
+Clear purpose
+-------------
+
+Metadata should only be present, if it has a purpose. It should not
+be included just because it might be useful in future, as if no one is
+using it, it is unlikely to be properly maintained.
+
+There should be just enough metadata to describe the data sufficiently.
+
+Nothing misleading or unnecessarily restrictive
+-----------------------------------------------
+
+Metadata that is misleading, or simply wrong, should not be present.
+
+Also, metadata that restricts wider use should be avoided, if possible.
+
+Referencing more detailed external documentation
+------------------------------------------------
+
+Ideally, the metadata should support the ability to 'refer out'
+to further information about the data to avoid 'overloading'
+the files with metadata.
+This makes it easier to provide more detail of the provenance of the data
+and the post-processing applied
+(the 'where' metadata described in the earlier note).
+
+One approach to help support this that has been partially adopted within
+IMPROVER is the use of name-spacing for new (non-CF) attributes.
+This has two purposes:
+
+ * It clearly separates the attribute from CF standard attributes,
+   identifying it as belonging to a separate metadata convention.
+ * It provides the potential for this attribute to be 'resolved' to a
+   definition held in an external registry in the future
+
+Namespacing is implemented using a double underscore to maintain CF conformance
+whilst clearly identifying the namespace part of the attribute name.
+For example, the "ssp" is used as statistical post-processing namespace,
+with the attributes in this namespace prefixed by ``spp__``.
+At present this is only used for a single attribute ``ssp__relative_to_threshold``,
+which takes one of the four values:
+
+    * ``greater_than`` 
+    * ``greater_than_or_equal_to``
+    * ``less_than`` 
+    * ``less_than_or_equal_to`` 
+
+The other namespace used is "mosg", which is used to indicate
+a Met Office standard grid attribute,
+which despite the name can, and should, generally be used.
```

Comment:
This is currently quite a long sentence, the following edit might aid readability.

```suggestion
Despite the name this can, and generally should, be used.
```

Reply:
Done